extensions/amp-story/1.0/amp-story-access.css

```diff
@@ -110,9 +109,17 @@ amp-story-access.i-amphtml-story-access-visible::before {
     flex-direction: column !important;
     margin: 0 !important;
     max-height: 80vh !important;
-    min-height: 40vh !important;
+    min-height: 20vh !important;
```

Comment:
All of these viewport-based sizes should use e.g. `calc(80 * --i-amphtml-story-vh)` since they are relative to the size of the amp-story-page and not the viewport on desktop.